lib/recipe/widgets/probe_list.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gibsonify/recipe/recipe.dart';
+
+class ProbeList extends StatelessWidget {
+  final int recipeIndex;
+
+  const ProbeList({Key? key, required this.recipeIndex}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<RecipeBloc, RecipeState>(builder: (context, state) {
+      return ListView.builder(
+          padding: const EdgeInsets.all(2.0),
+          itemCount: state.recipes[recipeIndex].probes.length,
+          itemBuilder: (context, index) {
+            return Card(
+                child: InkWell(
+                    onLongPress: () => {
+                          showDialog<String>(
+                              context: context,
+                              builder: (BuildContext context) => DeleteProbe(
+                                  recipe: state.recipes[recipeIndex],
+                                  probe:
+                                      state.recipes[recipeIndex].probes[index]))
+                        },
```

Comment:
On desktop this works, but on the Android emulator I cannot delete the probe. Long-pressing just starts to edit the text. We can deal with this later I guess.

Reply:
This needs to be fixed in the future if the same happens on non-emulated Android.

Reply:
I realise this till you mentioned it - I have instead switched to using a slidable widget for item deletion which is more consistent with modern apps

Reply:
On Android, I personally lean a little towards the long-press option when having to decide between sliding or long-pressing to delete, but again, the sliding fixes this issue so let's go ahead with that now and ultimately it'll be based on the feedback we gather what we end up using.

Reply:
In my experience, especially for less technical users, having an explicitly visible button of some sort to trigger actions for some object (such as a `Card` or `ListTile`) usually results in less confusion when trying to achieve some functionality compared to non-explicitly visible methods like long-pressing or sliding.

For example, trailing buttons such as a triple-dot that bring a pop up with further actions are not so visually intrusive, but achieve this, e.g. in Bitwarden:
![Bitwarden](https://bitwarden.com/help/images/getting-started/bitwarden-ios-trio.png).

But agaaaaain, this is best left for later 😂.